docs/validators/notation.md

```diff
@@ -0,0 +1,253 @@
+
+```yaml title="charts/connaisseur/values.yaml"
+- name: mynotation
+  type: notation
+  trustRoots:
+    - name: default
+      cert: |
+        -----BEGIN CERTIFICATE-----
+        MIIDrjCCApagAwIBAgIUfA/t/J6eINSu566aAozkOjQKey4wDQYJKoZIhvcNAQEL
+        BQAwbDELMAkGA1UEBhMCREUxDzANBgNVBAgMBkJlcmxpbjEPMA0GA1UEBwwGQmVy
+        # ... rest of your certificate ...
+        -----END CERTIFICATE-----
+```
+
+In `.application.policy`, add a pattern to match your repository:
+
+```yaml title="charts/connaisseur/values.yaml"
+- pattern: "docker.io/myrepo/*:*"  # YOUR REPOSITORY
+  validator: mynotation
+```
+
+After installation, Connaisseur will verify your images against the configured certificates:
+
+```bash
+helm install connaisseur helm --atomic --create-namespace --namespace connaisseur
+```
+
+### Testing validation
+
+You can test the validation by deploying pods with signed and unsigned images:
+
+```bash
+# This should succeed if the image is properly signed
+kubectl run signed-app --image=docker.io/myrepo/myimage:signed
+
+# This should fail if the image is not signed or signed with a different certificate
+kubectl run unsigned-app --image=docker.io/myrepo/myimage:unsigned
+```
+
+## Configuration options
+
+`.application.validators[*]` in `charts/connaisseur/values.yaml` supports the following keys for Notation (refer to [basics](../basics.md#validators) for more information on default keys):
+
+| Key | Default | Required | Description |
+| - | - | - | - |
+| `name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
+| `type` | - | :heavy_check_mark: | `notation`; the validator type must be set to `notation`. |
+| `trustRoots[*].name` | - | :heavy_check_mark: | See [basics](../basics.md#validators). |
+| `trustRoots[*].cert` | - | :heavy_check_mark: | X.509 certificate in PEM format used for signature verification. This should be the root signing certificate or a trusted CA certificate. |
+| `trustRoots[*].tsCert` | - | - | X.509 certificate in PEM format used for timestamp counter signature verification. |
+| `auth.` | - | - | Authentication credentials for registries with restricted access (e.g. private registries or rate limiting). See additional notes [below](#authentication). |
+| `auth.secretName` | - | - | Name of a Kubernetes secret in Connaisseur namespace that contains [dockerconfigjson](https://kubernetes.io/docs/concepts/configuration/secret/#docker-config-secrets) for registry authentication. See additional notes [below](#dockerconfigjson). |
+| `cert` | - | - | A TLS certificate in PEM format for private registries with self-signed certificates. |
+
+`.application.policy[*]` in `charts/connaisseur/values.yaml` supports the following additional keys for Notation (refer to [basics](../basics.md#image-policy) for more information on default keys):
+
+| Key | Default | Required | Description |
+| - | - | - | - |
+| `with.trustRoot` | - | :heavy_check_mark: | Name of the trust root to use for verification. |
```

Comment:
I think the required checkmark is wrong here and for other validators, as if the trustroot name is default, this is picked, or am I mistaken?